### ISSUE TITLE
Error: Point#multiply: invalid scalar, expected positive integer In Express Middleware

### ISSUE THREAD

**Issue by Yamboy1** [discord/discord-interactions-js#5]
When trying to save the "Interactions Endpoint URL" using the Express Middleware, I get this error:
```
/var/home/dobby/n/bin/node /var/home/dobby/interactions/index.js
Example app listening at http://localhost:3000
/var/home/dobby/interactions/node_modules/noble-ed25519/index.js:255
            throw new Error('Point#multiply: invalid scalar, expected positive integer');
                  ^

Error: Point#multiply: invalid scalar, expected positive integer
    at ExtendedPoint.multiply (/var/home/dobby/interactions/node_modules/noble-ed25519/index.js:255:19)
    at Object.verify (/var/home/dobby/interactions/node_modules/noble-ed25519/index.js:610:35)
    at processTicksAndRejections (node:internal/process/task_queues:93:5)

Process finished with exit code 1
```

The code looks like this:
```js
const express = require("express");
const { verifyKeyMiddleware } = require("discord-interactions");

const app = express();
const port = 3000;

app.post("/", verifyKeyMiddleware("<CLIENT_PUBLIC_KEY>"), (req, res) => {
  if (req.body.json()["type"] === 1) {
    res.json({ "type": 1 });
  }
})

app.listen(port, () => {
  console.log(`Example app listening at http://localhost:${port}`)
})

```

I'm also using ngrok for tunnelling the webhook url.

**Comment by Yamboy1** [comment]
Also note that i'm using node v15

**Comment by typpo** [comment]
Looks like some of the recent typescript cleanup broke the way signature and timestamp were fetched from headers.  Fixed in 38c20186df1a18d1730eca8d4b0b391c8162d3c7, if you upgrade to 1.2.1 it should be good now
